apps/landingPage/components/Info/ResponsiveText.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import React from 'react';
+import { Text } from '@kiwicom/orbit-components/lib/';
+import styled from 'styled-components';
+
+import { BREAKPOINTS } from '../../mediaQueriesConfig';
+
+const text =
+  "PLACEHOLDER!!!!!You've had a look at Tequila, but you've preffered yourTequila Tequila mixed with something else. Just grabMargarita.You've had a look at Tequila, but you've always prefferedyour Tequila mixed with something else.u've had a look at Tequ";
+
+export default function ResponsiveText() {
+  return (
+    <WrapperText>
+      <Mobile>
+        <Text size="large" align="left">
+          {text}
+        </Text>
+      </Mobile>
+      <Desktop>
+        <Text size="large" align="center">
+          {text}
+        </Text>
+      </Desktop>
+    </WrapperText>
+  );
+}
+
+const Mobile = styled.div`
+  @media (min-width: ${BREAKPOINTS.BIG_MOBILE}px) {
+    display: none;
+  }
+`;
+
+const Desktop = styled.div`
+  display: none;
+  @media (min-width: ${BREAKPOINTS.BIG_MOBILE}px) {
+    display: flex;
+  }
+`;
```

Comment:
These `Mobile` and `Desktop` styled components look the same as in the `ButtonInfo.js` so probably we can extract them and reuse. 🤔

Reply:
Should I extract them together with `Tablet` into `landingPage/components/styles/index.js` ?

Reply:
🤔 I didn't realize that. So, in that case, it is OK.